calculate/langauges.go

```diff
@@ -17,6 +17,7 @@ var languageToExtensions = map[Language][]string{
 	"rust":       {"rs"},
 	"scala":      {"scala", "sc"},
 	"php":        {"php", "phtml", "php3", "php4", "php5", "php7", "phps", "pht", "phar"},
+	"fortran":    {"f", "for", "f77", "f90", "f95", "f2k", "f03", "f03p", "f08", "f08p", "f15", "f20", "f18", "f2k", "f2003", "f2008", "f2015", "f2018", "fpp", "ftn", "f05", "F", "FOR", "F77", "F90", "F95", "F03", "F08", "F15", "F18", "F2K", "F2003", "F2015", "F2008", "F2018"},
```

Comment:
Looking at the other examples, all of them are lower case, I wonder if this should be distinct lower case values and the mechanism knows to be case insensitive (should be tested)